airbyte-integrations/connectors/source-survey-sparrow/source_survey_sparrow/survey_sparrow.yaml

```diff
@@ -0,0 +1,111 @@
+version: "0.1.0"
+
+definitions:
+  page_size: 50
+  selector:
+    extractor:
+      field_pointer: ["data"]
+  requester:
+    url_base: "https://eu-api.surveysparrow.com/v3"
```

Comment:
in the docs SurveySparrow says api.surveysparrow can you check maybe other servers and locations?

Reply:
Just push the change, sorry for that! :-) I thought I already committed